components/context-dialog/src/context-dialog.ts

```diff
@@ -197,9 +200,13 @@ export class DtContextDialog extends _DtContextDialogMixinBase
     private _viewContainerRef: ViewContainerRef,
     private _changeDetectorRef: ChangeDetectorRef,
     private _focusTrapFactory: FocusTrapFactory,
+    private _elementRef: ElementRef<HTMLElement>,
```

Comment:
You have to make it optional here this would brake the api and annotate with a deprecation to be non optional